modules/freetype/include/opencv2/freetype.hpp

```diff
@@ -99,11 +99,11 @@ If you want to draw small glyph, small is better.
 
 The function putText renders the specified text string in the image. Symbols that cannot be rendered using the specified font are replaced by "Tofu" or non-drawn.
 
-@param img Image.
+@param img Image. (Only 8UC3 image is supported.)
 @param text Text string to be drawn.
 @param org Bottom-left/Top-left corner of the text string in the image.
 @param fontHeight Drawing font size by pixel unit.
-@param color Text color.
+@param color Text color.(Only 3ch 8bit Scalar is supported.)
```

Comment:
> 8bit

Scalar values are `double`s by design.
I believe we should mention used range, like 0-255, here instead.

Reply:
Thank you very much! 

Because freetype2 wrapper module requests that img is  8UC3, I reconsider this limitation of color is very clear. 
  - first 3 channels of 'color' are used, 4th channel of it is not. 
  - The range of element are 0-255. 

This change is not necessary.  I cancel adding comment.